src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -30,6 +86,17 @@ public static void AddAppSecrets(this IServiceCollection services, IConfiguratio
             {
                 services.AddSingleton<ISecrets, SecretsLocalAppSI>();
             }
+
+            // Set up application insights
+            string applicationInsightsKey = env.IsDevelopment() ?
```

Comment:
Included the AI setup here as well.. 
Considered including the swagger setup as well, but that would stop app developer from changing the path to the swagger, so I left it as is..